### PR TITLE
Polish translation updated

### PR DESCRIPTION
**Pull request by napcok**

### CODE REVIEW

File: config/translations/lxqt-config-notificationd_pl.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="en_US">
```
Comment (palinek) [inline config/translations/lxqt-config-notificationd_pl.ts]: Didn't you mean `pl_PL` here?

File: src/translations/lxqt-notificationd_pl_PL.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="en_US">
```
Comment (palinek) [inline src/translations/lxqt-notificationd_pl_PL.ts]: ...and here too

File: config/translations/lxqt-config-notificationd_pl_PL.desktop [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # Translations
-GenericName[pl_PL]=Konfiguracja wyglądu LxQt
-Name[pl_PL]=Konfiguracja wyglądu LxQt
+GenericName[pl_PL]=Powiadomienia
+Name[pl_PL]=Powiadomienia
 Comment[pl_PL]=Konfiguruj powiadomienia freedesktop w LxQt
```
Comment (palinek) [inline config/translations/lxqt-config-notificationd_pl_PL.desktop]: IMHO...official name is `LXQt`, please, change it that way.

**Comment by napcok** [comment]
Opss...
Can you reject this commit please?
I will submit fixed one soon.

**Comment by palinek** [comment]
You can change/update the commit as you wish (have a look here for example: https://www.atlassian.com/git/tutorials/rewriting-history/git-commit--amend)

**Comment by paulolieuthier** [comment]
Closing.

**Pull request closed without merging**